app/src/main/java/com/dashxdemo/app/feature/login/LoginActivity.kt

```diff
@@ -3,6 +3,7 @@ package com.dashxdemo.app.feature.login
 import android.content.Intent
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import com.dashxdemo.app.R
```

Comment:
@sravank9 There's an unused import here, after you made the PR change. Make sure you take care of this in the next PR. cc @kiranattilicho